js/components/widgets/thumbnailer.vue

```diff
@@ -127,20 +125,20 @@ export default {
         /**
          * Adjust the preview given the cropper position and size
          */
-        preview: function(coords) {
-            let bounds = this.Jcrop.getBounds(),
-                w = bounds[0],
-                h = bounds[1],
-                // Temp fix until https://github.com/vuejs/vue/issues/1697 is merged
-                // containers = this.$els.preview_containers;
-                containers = [...this.$el.querySelectorAll('.preview-container')];
+        preview(coords) {
+            const bounds = this.Jcrop.getBounds();
+            const w = bounds[0];
+            const h = bounds[1];
+            // Temp fix until https://github.com/vuejs/vue/issues/1697 is merged
+            // containers = this.$els.preview_containers;
+            const containers = [...this.$el.querySelectorAll('.preview-container')];
 
             containers.forEach(function(el) {
```

Comment:
I'm not sure we are at that level of performances versus consistency. But fair enough.